projects/ngrx.io/content/guide/component-store/lifecycle.md

```diff
@@ -14,6 +14,18 @@ Currently, Angular provides initializer tokens in a few areas. The `APP_INITIALI
 
 </div>
 
+<div class="alert is-important">
+
+**Note:** If you override the `ngOnDestroy` method in your component store, you need to call `super.ngOnDestroy()`. Otherwise a memory leak may occur.
```

Comment:
This would go better down at the bottom with the `OnDestroy` section.

Reply:
absolutely. i changed that.